experimental/examples/imageviewer/gradle.properties

```diff
@@ -13,5 +13,5 @@ kotlin.native.useEmbeddableCompilerJar=true
 kotlin.native.binary.memoryModel=experimental
 kotlin.version=1.8.0
 agp.version=7.1.3
-compose.version=1.4.0-alpha01-dev972
+compose.version=1.4.0-alpha01-dev975
```

Comment:
This version upgrade fixes crash https://github.com/JetBrains/compose-multiplatform/issues/2845